types/json-rpc-error/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for json-rpc-error 2.0
+// Project: https://github.com/claudijo/json-rpc-error
+// Definitions by: Leonid Logvinov <https://github.com/LogvinovLeon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+export class InternalError extends Error {
+    constructor(err: Error | string);
+}
+export class MethodNotFound extends Error {
+    constructor();
+}
```

Comment:
This seems incorrect. 
The usage from https://github.com/claudijo/json-rpc-error#jsonrpcerror shows this package is a constructor for ```JsonRpcError```